docker-compose.yml

```diff
@@ -32,4 +35,10 @@ services:
   dev-xdebug:
     <<: *dev
     build: docker/xdebug
-
+  exasol:
+    image: exasol/docker-db:latest
+    privileged: true
```

Comment:
mohl bys tam prosim pridat ten limit na 1 cpu a 2GB pameti. Tak mi to cca frci

Reply:
přihodil sem `docker-compose.limits.yml` a přidal o tom poznámku do readme